cli/integration_tests/lockfile_aware_caching/npm.t

```diff
@@ -42,16 +42,16 @@ Both a and b should have a cache miss since npm isn't implemented
   \xe2\x80\xa2 Packages in scope: a (esc)
   \xe2\x80\xa2 Running build in 1 packages (esc)
   \xe2\x80\xa2 Remote caching disabled (esc)
-  a:build: cache miss, executing [0-9a-f]+ (re)
+  a:build: cache hit, replaying output [0-9a-f]+ (re)
```

Comment:
Should update the title here (line 38 outside of this diff)
> Both a and b should have a cache miss since npm isn't implemented

No longer the case